ui/components/app/add-network/add-network.js

```diff
@@ -43,20 +43,20 @@ const AddNetwork = () => {
   const t = useContext(I18nContext);
   const dispatch = useDispatch();
   const history = useHistory();
-  const frequentRpcList = useSelector(getFrequentRpcListDetail);
+  const networkconfigurations = useSelector(getNetworkConfigurations);
```

Comment:
Nit: lower-case C here not aligned with our usual naming conventions

```suggestion
  const networkConfigurations = useSelector(getNetworkConfigurations);
```

Reply:
fixed here: https://github.com/MetaMask/metamask-extension/pull/17421/commits/ef2df393c9838e83875c78b10cc411b0cdb6c76c